docs/ref/settings.txt

```diff
@@ -687,13 +689,16 @@ When :setting:`USE_TZ` is ``False``, it is an error to set this option.
     third-party systems connect to the same database and expect to find
     datetimes in local time, then you must set this option.
 
-* If the database backend supports time zones (e.g. PostgreSQL), the
-  ``TIME_ZONE`` option is very rarely needed. It can be changed at any time;
-  the database takes care of converting datetimes to the desired time zone.
+* If the database backend supports time zones (e.g., PostgreSQL), then the
```

Comment:
I personally think that this comma does belong here, though I see that in some other places this pattern is used. Though from 981 "e.g." in the `django` folder, 41 are followed by a comma, and from 457 in the `docs` folder, 84 have the following comma.

Is this an English [from a specific region?] style pattern that I'm missing?

```suggestion
* If the database backend supports time zones (e.g. PostgreSQL), then the
```

Reply:
I don't see that pattern but it could just be my country🤷‍♂️